python/sdist/amici/sbml_import.py

```diff
@@ -851,6 +863,25 @@ def add_d_dt(
                 'dt': d_dt,
             }
 
+    @log_execution_time('processing SBML annotations', logger)
+    def _process_annotations(self) -> None:
+        # Remove all parameters (and corresponding rules)
+        # for which amici:discard is set
```

Comment:
```suggestion
        """
        Remove all parameters (and corresponding rules)
        for which amici:discard is set
        """
```

Reply:
In theory that is only one particular thing this function could do. It just happens it is the only thing it does at the moment. I made a more general doc-string.